CHANGE-NOTES.html

```diff
@@ -1,15 +1,8 @@
-Added support for IntelliJ 2020.3.
-<br/>
-<br/>
 Bug fixes/improvements:
 <ul>
-  <li>"Slide In Branch Below Current Branch" action is now hidden for unmanaged branches</li>
-  <li>Fixed faulty .git/machete file parsing when the last line has only whitespace</li>
-  <li>Improved UX of Override Fork Point dialog</li>
-  <li>Allowed fast forwarding parent when a branch is connected to the parent with a yellow edge</li>
-  <li>Fixed font size dependent graph scaling</li>
-  <li>Fixed incorrect indication of newly created branch after pulling its parent as merged</li>
-  <li>Allowed checkout and rebase on detached state</li>
-  <li>Improved fork point inference</li>
-  <li>Added rediscover suggestion after long Git Machete non-use time</li>
+  <li>Add link to Open Machete File action from discovery success notification</li>
+  <li>Improve stability of fast-forward pull action</li>
+  <li>Prohibit re-checking out the currently checked out with double click</li>
+  <li>Rename "fast-forward parent to match branch" action to "fast-forward merge branch to parent"</li>
+  <li>Treat develop branch as root in discovery, even if master branch is present</li>
```

Comment:
Shouldn't new improvements/fixes be added to previous one as it's a patch release?

Reply:
Right, added a new section for 0.8.2's changes